doc/mgr/orchestrator.rst

```diff
@@ -211,6 +230,9 @@ You can query the state of the operation with::
 
 When no PGs are left on the osd, it will be decommissioned and removed from the cluster.
 
+.. note::
```

Comment:
s/``--all-available-devices``,/``all-available-devices``/ (remove the comma after " ``all-available-devices`` ")